src/apps/material/material.dev.tsx

```diff
@@ -663,6 +663,11 @@ export default {
       defaultValue: "Email",
       control: { type: "text" }
     },
+    orderDigitalCopyFeedbackErrorMissingMunicipalityagencyidText: {
```

Comment:
This needs to be added to dpl-cms as well, right?

Reply:
https://reload.zulipchat.com/#narrow/stream/240325-DDF/topic/Open.20Order.20tekst.20strenge

Reply:
?